sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1337,7 +1337,7 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 			case 200:
 				currentVersion = 200;
 				getDao(Symptoms.class).executeRaw("ALTER TABLE symptoms ADD COLUMN coughWithSputum varchar(255);");
-				getDao(Symptoms.class).executeRaw("ALTER TABLE symptoms ADD COLUMN coughWithHeamoptysis varchar(255);");
+				getDao(Symptoms.class).executeRaw("ALTER TABLE symptoms ADD COLUMN coughWithHemoptysis varchar(255);");
```

Comment:
Revert this change because it will break existing servers. You need to write a migration script instead that renames the column. Since SQLite unfortunately doesn't support renaming columns, you'll probably have to re-create the whole symptoms table by creating a temporary table as a copy of the existing symptoms table and transfering the values between those tables. We've done this already before in the mobile app for other tables, e.g. in version number 140 for samples.

Reply:
Well noted @MateStrysewske, should I leave this issue for your developer to finish?

Reply:
We don't really have resources to work on this right now; since it's not an extremely urgent issue, I'd appreciate if you could work on it; otherwise, we'll take care of it at some point in the future.